iOS/Flows/Conversation/ConversationViewControllerDataSource.swift

```diff
@@ -160,6 +185,78 @@ class ConversationCollectionViewDataSource: CollectionViewDataSource<Conversatio
             }
         }
     }
+
+    /// Updates the datasource with the passed in array of conversation changes.
+    func update(with changes: [ListChange<Conversation>],
+                conversationController: ConversationListController,
+                collectionView: UICollectionView) async {
```

Comment:
Copying the apply changes function to work with conversation changes. It's not elegant. Refactor would have one function.